### ISSUE TITLE
Add Cloud RMs

### ISSUE THREAD

**Issue by natolambert** [allenai/reward-bench#173]
See @zankner's repo https://github.com/zankner/CLoud, RM's that think out loud!

**Comment by scottsuk0306** [comment]
Hi @natolambert, I recently trained a generative RM ([prometheus-eval/prometheus-RM-Llama-8B-v1.0](https://huggingface.co/prometheus-eval/prometheus-RM-Llama-8B-v1.0)) based on the CLoud code base and l think the inference using the huggingface transformers can be easily integrated to existing code of reward-bench. Can I try working on this?

**Comment by natolambert** [comment]
@scottsuk0306 yes please! I've been curious for a bit and we're building new datasets now

**Comment by zankner** [comment]
Hi sorry realized I never followed up on this. @scottsuk0306 @natolambert we support HF inference in the CLoud repo which I can also add ([hf-inference](https://github.com/zankner/CLoud/blob/fac417e2c0f45fab9083bf9b77066de490e8288c/cloud/model.py#L105)). However, HF generate is extremely slow. More recently we added [vllm-support](https://github.com/zankner/CLoud/blob/main/cloud/inference/vllm_llama_cloud.py) but I'm not sure whether that would be easy to support in your repo, if you have any thoughts.

**Comment by natolambert** [comment]
We could try running Cloud RM with the "generative" pipeline, which is a bit different.
Otherwise, I feel like it isn't needed to use VLLM, unless we can figure out how to wrap it in the same abstraction -- which actually may be possible, because we just do some general model loading.

Either way, curious where we end up.

**Comment by natolambert** [comment]
Closing as stale.
